README.md

```diff
@@ -25,6 +25,8 @@ This repo contains examples in each language supported by Amazon Keyspaces. This
 - [Ruby Examples](https://github.com/aws-samples/amazon-keyspaces-examples/tree/main/ruby)
 - [Python Examples](https://github.com/aws-samples/amazon-keyspaces-examples/tree/main/python)
 - [Scala Examples](https://github.com/aws-samples/amazon-keyspaces-examples/tree/main/scala)
+- [SageMaker Examples](https://github.com/aws-samples/amazon-keyspaces-examples/tree/main/SageMaker)
```

Comment:
The format is to break these down by <language>/<driver version>/<project name>
So this should be put under python/datastax-v3/sagemaker/